src/libs/ModifiedExpenseMessage.ts

```diff
@@ -143,21 +159,7 @@ function getForReportAction(reportAction: ReportAction): string {
         );
     }
 
-    const hasModifiedCreated = reportActionOriginalMessage && 'oldCreated' in reportActionOriginalMessage && 'created' in reportActionOriginalMessage;
-    if (hasModifiedCreated) {
-        // Take only the YYYY-MM-DD value as the original date includes timestamp
-        let formattedOldCreated: Date | string = new Date(reportActionOriginalMessage?.oldCreated ? reportActionOriginalMessage.oldCreated : 0);
-        formattedOldCreated = format(formattedOldCreated, CONST.DATE.FNS_FORMAT_STRING);
-        buildMessageFragmentForValue(
-            reportActionOriginalMessage?.created ?? '',
-            formattedOldCreated,
-            Localize.translateLocal('common.date'),
-            false,
-            setFragments,
-            removalFragments,
-            changeFragments,
-        );
-    }
+    buildMessageFragmentForCreated(setFragments, removalFragments, changeFragments, reportActionOriginalMessage);
```

Comment:
Q: what was the need for extracting this logic into a function? I can't see this function used more than once so I'm not sure it's necessary

Reply:
I just extracted the code block responsible for building the message for the property `created` to this function. My idea was to decrease the complexity of the parent function `getForReportAction`, but if you disagree I can move the code from the `buildMessageFragmentForCreated` function and put it here.

Reply:
Yeah I think as that logic is not really reusable there's no need to extract it to a function. Thanks!

Reply:
Sure I understood your points, I'll change here.

Reply:
Done: [326f61d](https://github.com/Expensify/App/pull/33688/commits/326f61db384de6570d3a0fe0708acfc878ce1760)